src/app/auth/register/page.tsx

```diff
@@ -1,20 +1,36 @@
 import Link from 'next/link';
+import { redirect } from 'next/navigation';
+import { getServerSession } from 'next-auth';
 
-import { RegisterForm } from './form';
+import { RegisterForm } from './RegisterForm';
 
+import { authOptions } from '@/app/api/auth/[...nextauth]/route';
 import { createUser } from '@/services/userService';
 
 export default async function Page() {
+    const session = await getServerSession(authOptions);
+    if (session) {
+        redirect('/');
```

Comment:
We can handle this in middleware. Let's do it later. [Ticket](https://tsoibet.atlassian.net/browse/KAADO-55)